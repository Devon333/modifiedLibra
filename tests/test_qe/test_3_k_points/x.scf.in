&CONTROL
  calculation = 'scf',
  dt = 20.67055,
  nstep = 50,
  pseudo_dir = './',
  outdir = './',
  prefix = 'x',
  disk_io = 'low',
/

&SYSTEM
  ibrav = 0,
  celldm(1) = 1.89,
  nat = 6,
  ntyp = 2,
  nspin = 1,
  nbnd = 10,
  ecutwfc = 2,
  tot_charge = 0.0,
  occupations = 'smearing',
  smearing = 'gaussian',
  degauss = 0.005,
  nosym = .true.,

/

&ELECTRONS
  electron_maxstep = 300,
  conv_thr = 1.D-5,
  mixing_beta = 0.45,
/

&IONS
  ion_dynamics = 'verlet',
  ion_temperature = 'andersen',
  tempw = 300.00 ,
  nraise = 1,
/


ATOMIC_SPECIES
 C  12.01  C.pbe-n-kjpaw_psl.0.1.UPF
 H  1.008  H.pbe-kjpaw_psl.0.1.UPF


K_POINTS automatic
4 1 1  0 0 0
                               
CELL_PARAMETERS
    20.0000000    0.0000000    0.0000000
     0.0000000   20.0000000    0.0000000
     0.0000000    0.0000000   20.0000000
 
ATOMIC_POSITIONS (alat)
C         -5.04672        2.31763        0.01782
C         -3.71548        2.30894       -0.00587
H         -5.60642        1.54073       -0.49255
H         -5.57772        3.10164        0.54761
H         -3.18448        1.52493       -0.53565
H         -3.15579        3.08584        0.50451

