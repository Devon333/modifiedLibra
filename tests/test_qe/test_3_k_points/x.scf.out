
     Program PWSCF v.5.2.0 starts on 14Jul2016 at 19:15: 3 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Serial version
     Waiting for input...
     Reading input from standard input
Warning: card &IONS ignored
Warning: card   ION_DYNAMICS = 'VERLET', ignored
Warning: card   ION_TEMPERATURE = 'ANDERSEN', ignored
Warning: card   TEMPW = 300.00 , ignored
Warning: card   NRAISE = 1, ignored
Warning: card / ignored
     Message from routine read_cards :
     DEPRECATED: no units specified in CELL_PARAMETERS card

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
               file C.pbe-n-kjpaw_psl.0.1.UPF: wavefunction(s)  2P renormalized
               file H.pbe-kjpaw_psl.0.1.UPF: wavefunction(s)  1S renormalized

     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         901     901    253                20479    20479    3071



     bravais-lattice index     =            0
     lattice parameter (alat)  =       1.8900  a.u.
     unit-cell volume          =   54010.1520 (a.u.)^3
     number of atoms/cell      =            6
     number of atomic types    =            2
     number of electrons       =        12.00
     number of Kohn-Sham states=           10
     kinetic-energy cutoff     =       2.0000  Ry
     charge density cutoff     =       8.0000  Ry
     convergence threshold     =      1.0E-05
     mixing beta               =       0.4500
     number of iterations used =            8  plain     mixing
     Exchange-correlation      =  SLA  PW   PBX  PBC ( 1  4  3  4 0 0)

     celldm(1)=   1.890000  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  20.000000   0.000000   0.000000 )  
               a(2) = (   0.000000  20.000000   0.000000 )  
               a(3) = (   0.000000   0.000000  20.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  0.050000  0.000000  0.000000 )  
               b(2) = (  0.000000  0.050000  0.000000 )  
               b(3) = (  0.000000  0.000000  0.050000 )  


     PseudoPot. # 1 for  C read from file:
     ./C.pbe-n-kjpaw_psl.0.1.UPF
     MD5 check sum: c849119b86527ef673bbe9c4f1f1b6ab
     Pseudo is Projector augmented-wave + core cor, Zval =  4.0
     Generated using "atomic" code by A. Dal Corso  v.5.0.2 svn rev. 9415
     Shape of augmentation charge: BESSEL
     Using radial grid of 1073 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
     Q(r) pseudized with 0 coefficients 


     PseudoPot. # 2 for  H read from file:
     ./H.pbe-kjpaw_psl.0.1.UPF
     MD5 check sum: 7353169556fa043aa23d7d2f4b8a0dcf
     Pseudo is Projector augmented-wave, Zval =  1.0
     Generated using "atomic" code by A. Dal Corso  v.5.0.2 svn rev. 9415
     Shape of augmentation charge: PSQ
     Using radial grid of  929 points,  2 beta functions with: 
                l(1) =   0
                l(2) =   0
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
        C              4.00    12.01000      C( 1.00)
        H              1.00     1.00800      H( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           C   tau(   1) = (  -5.0467200   2.3176300   0.0178200  )
         2           C   tau(   2) = (  -3.7154800   2.3089400  -0.0058700  )
         3           H   tau(   3) = (  -5.6064200   1.5407300  -0.4925500  )
         4           H   tau(   4) = (  -5.5777200   3.1016400   0.5476100  )
         5           H   tau(   5) = (  -3.1844800   1.5249300  -0.5356500  )
         6           H   tau(   6) = (  -3.1557900   3.0858400   0.5045100  )

     number of k points=     3  gaussian smearing, width (Ry)=  0.0050
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.5000000
        k(    2) = (   0.0125000   0.0000000   0.0000000), wk =   1.0000000
        k(    3) = (  -0.0250000   0.0000000   0.0000000), wk =   0.5000000

     Dense  grid:    20479 G-vectors     FFT dimensions: (  36,  36,  36)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         0.40 Mb     (    2594,   10)
        NL pseudopotentials             0.95 Mb     (    2594,   24)
        Each V/rho on FFT grid          0.71 Mb     (   46656)
        Each G-vector array             0.16 Mb     (   20479)
        G-vector shells                 0.00 Mb     (     243)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         1.58 Mb     (    2594,   40)
        Each subspace H/S matrix        0.02 Mb     (      40,   40)
        Each <psi_i|beta_j> matrix      0.00 Mb     (      24,   10)
        Arrays for rho mixing           5.70 Mb     (   46656,    8)

     Check: negative/imaginary core charge=   -0.000116    0.000000

     Initial potential from superposition of free atoms
     Check: negative starting charge=   -2.627254

     starting charge   11.99988, renormalised to   12.00000

     negative rho (up, down):  2.627E+00 0.000E+00
     Starting wfc are   12 randomized atomic wfcs

     total cpu time spent up to now is        1.0 secs

     per-process dynamical memory:    11.6 Mb

     Self-consistent Calculation

     iteration #  1     ecut=     2.00 Ry     beta=0.45
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  8.0

     negative rho (up, down):  1.125E+00 0.000E+00

     total cpu time spent up to now is        1.8 secs

     total energy              =     -31.53009928 Ry
     Harris-Foulkes estimate   =     -34.74042156 Ry
     estimated scf accuracy    <       5.30218486 Ry

     iteration #  2     ecut=     2.00 Ry     beta=0.45
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.0

     negative rho (up, down):  5.274E-01 0.000E+00

     total cpu time spent up to now is        2.4 secs

     total energy              =     -31.78700563 Ry
     Harris-Foulkes estimate   =     -32.18709799 Ry
     estimated scf accuracy    <       0.78583526 Ry

     iteration #  3     ecut=     2.00 Ry     beta=0.45
     Davidson diagonalization with overlap
     ethr =  6.55E-03,  avg # of iterations =  1.0

     negative rho (up, down):  7.528E-01 0.000E+00

     total cpu time spent up to now is        2.9 secs

     total energy              =     -31.77682144 Ry
     Harris-Foulkes estimate   =     -31.81406985 Ry
     estimated scf accuracy    <       0.08842601 Ry

     iteration #  4     ecut=     2.00 Ry     beta=0.45
     Davidson diagonalization with overlap
     ethr =  7.37E-04,  avg # of iterations =  3.0

     negative rho (up, down):  7.910E-01 0.000E+00

     total cpu time spent up to now is        3.5 secs

     total energy              =     -31.78153549 Ry
     Harris-Foulkes estimate   =     -31.78148534 Ry
     estimated scf accuracy    <       0.00172013 Ry

     iteration #  5     ecut=     2.00 Ry     beta=0.45
     Davidson diagonalization with overlap
     ethr =  1.43E-05,  avg # of iterations =  5.3

     negative rho (up, down):  7.970E-01 0.000E+00

     total cpu time spent up to now is        4.2 secs

     total energy              =     -31.78177007 Ry
     Harris-Foulkes estimate   =     -31.78168655 Ry
     estimated scf accuracy    <       0.00006897 Ry

     iteration #  6     ecut=     2.00 Ry     beta=0.45
     Davidson diagonalization with overlap
     ethr =  5.75E-07,  avg # of iterations =  4.3

     negative rho (up, down):  7.992E-01 0.000E+00

     total cpu time spent up to now is        4.8 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (  2553 PWs)   bands (ev):

   -40.5822 -26.2964 -15.9104 -11.3803  -8.9237  -8.7512  -3.1758  -3.1638
    -2.2477  -1.9449

          k = 0.0125 0.0000 0.0000 (  2561 PWs)   bands (ev):

   -40.5839 -26.3662 -15.8956 -11.3752  -8.9612  -8.8066  -3.1802  -3.1705
    -2.2665  -1.9417

          k =-0.0250 0.0000 0.0000 (  2594 PWs)   bands (ev):

   -40.7384 -26.5529 -15.9875 -11.4618  -9.0008  -8.9666  -3.2590  -3.1866
    -2.2818  -1.9415

     the Fermi energy is    -6.6891 ev

!    total energy              =     -31.78182392 Ry
     Harris-Foulkes estimate   =     -31.78177761 Ry
     estimated scf accuracy    <       0.00000115 Ry

     total all-electron energy =      -147.310010 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =     -71.08151611 Ry
     hartree contribution      =      31.95291631 Ry
     xc contribution           =      -7.84156769 Ry
     ewald contribution        =      28.31653655 Ry
     one-center paw contrib.   =     -13.12819300 Ry
     smearing contrib. (-TS)   =      -0.00000000 Ry

     convergence has been achieved in   6 iterations

     Writing output data file x.save

     init_run     :      0.51s CPU      0.52s WALL (       1 calls)
     electrons    :      3.65s CPU      3.89s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.11s CPU      0.11s WALL (       1 calls)
     potinit      :      0.27s CPU      0.28s WALL (       1 calls)

     Called by electrons:
     c_bands      :      1.56s CPU      1.66s WALL (       6 calls)
     sum_band     :      0.36s CPU      0.41s WALL (       6 calls)
     v_of_rho     :      0.47s CPU      0.46s WALL (       7 calls)
     newd         :      0.16s CPU      0.17s WALL (       7 calls)
     mix_rho      :      0.03s CPU      0.04s WALL (       6 calls)

     Called by c_bands:
     init_us_2    :      0.06s CPU      0.05s WALL (      39 calls)
     cegterg      :      1.45s CPU      1.60s WALL (      18 calls)

     Called by sum_band:
     sum_band:bec :     -0.00s CPU      0.00s WALL (      18 calls)
     addusdens    :      0.14s CPU      0.17s WALL (       6 calls)

     Called by *egterg:
     h_psi        :      1.19s CPU      1.20s WALL (      92 calls)
     s_psi        :      0.12s CPU      0.09s WALL (      92 calls)
     g_psi        :      0.01s CPU      0.03s WALL (      71 calls)
     cdiaghg      :      0.02s CPU      0.04s WALL (      89 calls)

     Called by h_psi:
     add_vuspsi   :      0.05s CPU      0.08s WALL (      92 calls)

     General routines
     calbec       :      0.13s CPU      0.13s WALL (     110 calls)
     fft          :      0.15s CPU      0.12s WALL (      90 calls)
     fftw         :      1.15s CPU      1.07s WALL (    1336 calls)
     davcio       :     -0.00s CPU      0.00s WALL (       3 calls)


     PAW routines
     PAW_pot      :      1.36s CPU      1.39s WALL (       7 calls)

     PWSCF        :     4.59s CPU         4.96s WALL


   This run was terminated on:  19:15: 8  14Jul2016            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
